.github/workflows/pull-request.yml

```diff
@@ -128,7 +128,7 @@ jobs:
     strategy:
       matrix:
         toolchain:
-          - 1.45.0 # MSRV
+          - 1.51.0 # min version with const generics
```

Comment:
integration tests should still use MSRV (only the unit tests run the macro tests)